### ISSUE TITLE
lh-map-tools: It should be possible to configure the colour used to highlight the marker

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/lh-vim#1]
```
Zdenek observed:
The only glitch is that those (default) marks '<<>>' (and only those marks,
the rest is fine) come now in "some light color" on white background, very
hard to read, while before they were nice blue on black bg which I liked.
Is there any new setting for colors, perhaps?

```

Original issue reported on code.google.com by `luc.herm...@gmail.com` on 22 Feb 2008 at 11:56

**Comment by GoogleCodeExporter** [comment]
```
I concur. Perhaps this issue's priority should be bumped up to high :) 
```

Original comment by `ngaloppo@gmail.com` on 22 Feb 2008 at 9:30

**Comment by GoogleCodeExporter** [comment]
```
Corrected in the branch v1.0.0 of map-tools.
Now the option [gb]:marker_highlight is expected to contain the parameters 
given to
the :hi command. 
By default, it values 'guibg=#0d0d0d ctermbg=darkgray' when &bg is 'dark', and
'guibg=#d0d0d0 ctermbg=lightgray' otherwise.
It can be overriden in the .vimrc. Setting it to an empty string disables this
highlighting.
```

Original comment by `luc.herm...@gmail.com` on 20 Mar 2008 at 12:36
- Changed state: **Fixed**
